app/repositories/repository_challenge.rb

```diff
@@ -162,7 +162,7 @@ def self.best_score_per_user(challenge_id)
           "created_at": { "$first": '$entries.created_at' },
           "script": { "$first": '$entries.script' },
           "comments": { "$first": '$entries.comments' },
-          "min_score": { "$min": '$entries.score'}
+          "min_score": { "$first": '$entries.score'}
```

Comment:
Hmm, for my own education.. Can you explain what the diff is between previous $min and $first here?

Reply:
At the end we get the same result. Just a little optimization and consistancy.

In details : 
Before this $group, we $sort by min_score
When calling $min, it needs to reads all min_score to find the 'min'
But because we sorted it, we can just take the $first

Reply:
Makes sense, ty!